src/OXRS_LCD.cpp

```diff
@@ -178,6 +277,56 @@ void OXRS_LCD::update(void)
       _last_lcd_trigger = 0L;
     }
   }
+  
+  // turn off rx LED if timed out
+  if (_last_rx_trigger)
+  {
+    if ((millis() - _last_rx_trigger) > RX_TX_LED_ON)
+    {
+      _set_mqtt_rx_led(0);
+      _last_rx_trigger = 0L;
+    }
+  }
+ 
+  // turn off tx LED if timed out
+  if (_last_tx_trigger)
+  {
+    if ((millis() - _last_tx_trigger) > RX_TX_LED_ON)
+    {
+      _set_mqtt_tx_led(0);
+      _last_tx_trigger = 0L;
+    }
+  }
+ 
+  // check if ethernet link status has changed
+  if (_ethernet)
+  {
+    // show IP and MAC if first time called
+    if (_first_call)
+    {
+      _show_ethernet();
+      _first_call = false;
+    }
+    uint8_t tmp = _ethernet->linkStatus();
+    if (tmp != _ethernet_link_status)
+    {
+      if (tmp != LinkON)
+      {
+        _show_IP(IPAddress(0,0,0,0), 0);
+      }
+      else
+      {
+        _show_IP(_ethernet->localIP(), 1);
+      }
+      _ethernet_link_status = tmp;
+    }
+  }
+
+  // check if wifi connection status has changed
+  if (_wifi)
```

Comment:
Similarly change this to `if (_wifi) _show_wifi();` and add a stub function for `_show_wifi()`.

Reply:
done